Manifest.toml

```diff
@@ -1,30 +1,59 @@
 # This file is machine-generated - editing it directly is not advised
 
-julia_version = "1.8.3"
+julia_version = "1.9.1"
```

Comment:
seems that you made the manifest with v1.9.1
but since the tests run on v1.8 it's better to do use v1.8.5 to make the manifest, just in case there is a conflict or something

Reply:
I can fix this; shall I?

Reply:
Yeah happy for you to fix, thanks